### PR TITLE
Add mongo (mongoid) support

### PR DESCRIPTION
**Pull request by maksimf**
#18

### CODE REVIEW

File: lib/capistrano-db-tasks/database.rb [hunk 1]
```diff
@@ -15,6 +16,21 @@ def postgresql?
       %w(postgresql pg).include? @config['adapter']
     end
 
+    def mongo?
```
Comment (numbata) [inline lib/capistrano-db-tasks/database.rb]: always `true`?

Comment (maksimf) [inline lib/capistrano-db-tasks/database.rb]: @numbata it seems i forgot to push last changes. Empty `mongo?` method and `require pry`. I'm sorry, will do

Comment (maksimf) [inline lib/capistrano-db-tasks/database.rb]: @numbata fixed

**Comment by maksimf** [comment]
@numbata Is anything else I should correct?

**Comment by mdebo** [comment]
@maksimf I have install your branch on my project. 
I have this error when i launch pull production db:

stdout: Failed global initialization: BadValue Invalid or no user locale set. Please ensure LANG and/or LC_* environment variables are set correctly.
I have checked the LANG env variable and it seems it is set correctly... Any idea?

Thanks

**Comment by maksimf** [comment]
@mdebo Can you provide the command you ran and full stack trace please?

**Comment by zedtux** [comment]
@maksimf for me it's almost working :

```
~/Developments/yellow • cap staging db:pull
Using airbrussh format.
Verbose output is being written to log/capistrano.log.
local {"sessions"=>{"default"=>{"database"=>"yellow_development", "hosts"=>["localhost:27017"], "options"=>nil}}, "options"=>nil}
Local database: yellow_development
Please enter answer (Are you sure you want to erase your local database with server database (y)es, (n)o  ? ): y
local {"sessions"=>{"default"=>{"database"=>"yellow_development", "hosts"=>["localhost:27017"], "options"=>nil}}, "options"=>nil}
00:00 db:local:sync
      01 cd /home/yellow/yellow/current && mongodump  -d yellow_staging  -o dump_2016-03-08-115530 && tar -zcvf dump_2016-03-08-115530.tar.gz dump_2016-03-08-115530/
    ✔ 01 yellow@staging.yellow.lu 16.001s
      Downloading dump_2016-03-08-115530.tar.gz 10.02%
      Downloading dump_2016-03-08-115530.tar.gz 20.03%
      ...
      Downloading dump_2016-03-08-115530.tar.gz 90.02%
      Downloading dump_2016-03-08-115530.tar.gz 100.0%
      leaving /home/yellow/yellow/current/dump_2016-03-08-115530.tar.gz on the server (add "set :db_remote_clean, true" to deploy.rb to remove)
x dump_2016-03-08-115530/
x dump_2016-03-08-115530/yellow_staging/
x dump_2016-03-08-115530/yellow_staging/test.metadata.json
...
2016-03-08T11:56:57.842+0100	building a list of collections to restore from yellow_development dir
2016-03-08T11:56:57.842+0100	Failed: error scanning filesystem: error reading db folder yellow_development: open yellow_development: no such file or directory
      false
      leaving ./dump_2016-03-08-115530 (specify :db_local_clean in deploy.rb to remove)
      Completed database import
```

The only thing I see in the code which doesn't looks good is that with your branch you have an SQL DB OR a NoSQL DB.
In our case, we have both. We should be able to pull both DB.

**Comment by zedtux** [comment]
For info, I did a `mongorestore dump_2016-03-08-115530/` and it worked.

**Comment by zedtux** [comment]
And also the following (as in the code `-d` is used) `mongorestore -d yellow_development  dump_2016-03-08-115530/yellow_staging`.

**Comment by zedtux** [comment]
The previous one failed after a while with an already existing object. Adding a `--drop` to the command helped.

**Pull request closed without merging**